src/main/java/com/github/soerxpso/xpspawners/manager/SpawnerManager.java

```diff
@@ -0,0 +1,66 @@
+package com.github.soerxpso.xpspawners.manager;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+
+import com.github.soerxpso.xpspawners.Spawner;
+
+public class SpawnerManager {
+	private HashMap<Long, ArrayList<Spawner>> spawners = 
+			new HashMap<Long, ArrayList<Spawner>>();
+	
+	public ItemStack convertSpawnerToItem(Block b) {
+		return new ItemStack(Material.MOB_SPAWNER);
```

Comment:
I guess it's not a big deal at this step if placing this spawner produces a pig spawner, yeah?